pulsar-broker/src/test/java/org/apache/pulsar/client/api/NonPersistentTopicTest.java

```diff
@@ -824,10 +825,12 @@ public void testMsgDropStat() throws Exception {
             stopBroker();
             startBroker();
             Consumer<byte[]> consumer = pulsarClient.newConsumer().topic(topicName).subscriptionName("subscriber-1")
-                    .receiverQueueSize(1).subscribe();
+                    .receiverQueueSize(1)
+                    .messageListener((c, msg) -> {}).subscribe();
```

Comment:
Why a dummy message listener is necessary here?

Reply:
It might not be necessary, but it will consume and acknowledge messages.

Reply:
My assumption is that it's a better simulation of a real application than a subscription that doesn't consume messages.

Reply:
Maybe it doesn't acknowledge by default... well, it will at least consume the messages.